x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -40,17 +40,15 @@ export const DETECTION_RULE_ALERTS_STATUS_API_CURRENT_VERSION = '1';
 export const DETECTION_RULE_RULES_API_CURRENT_VERSION = '2023-10-31';
 
 export const CLOUD_SECURITY_POSTURE_PACKAGE_NAME = 'cloud_security_posture';
-// TODO: REMOVE CSP_LATEST_FINDINGS_DATA_VIEW and replace it with LATEST_FINDINGS_INDEX_PATTERN
-export const CSP_LATEST_FINDINGS_DATA_VIEW = 'logs-cloud_security_posture.findings_latest-*';
+
+export const CDR_LATEST_FINDINGS_INDEX_PATTERN = 'logs-*finding*_latest-*';
+export const CDR_FINDINGS_DATA_VIEW_NAME = 'cloud-security-data-view';
```

Comment:
One more note on the naming - during the work on RFC I found it extremely confusing that we use the terms `finding`, `configuration`, and `misconfiguration` interchangeably. I proposed to use the terms `misconfiguration` and `vulnerability` consistently and have `finding`  only when we refer to an event that can be both misconfiguration and vulnerability (which is quite rare I believe atm). I think it follows the industry terminology better and reflects the current reality that we have the Findings page with two separate views for Misconfigurations and Vulnerabilities